src/components/Footer/Footer.css

```diff
@@ -41,13 +41,18 @@
   width: 100%;
 
   @media (--viewportMedium) {
-    max-width: 1056px;
     margin: 0 auto;
   }
 
   @media (--viewportLarge) {
+    max-width: 1104px;
```

Comment:
There's still mismatch (at least) on LandingPage with paddings. Topbar and SectionHero use 36px padding but content and Footer use 24px. That is pretty clear on one quite important breakpoint 1024px, which is common width on tablets.

So, I'd push this change a bit further and recommend paddings to be 36px starting from that break
![screenshot 2019-02-04 at 14 36 47](https://user-images.githubusercontent.com/717315/52211588-f9a22480-2892-11e9-83fd-31b063369cb8.png)
point.